### DISCUSSION TITLE
Adding support for Namron Z-Wave dimmer 2

### DISCUSSION THREAD

**Discussion by GD-Dal** [zwave-js/zwave-js#2515] [Q&A]
Hi.
I need to ask for help here.
I have recently bought a Namron Z-Wave dimmer 2, only to discover that it wasn't supported in Z-Wave JS

Home Assistant reports the device as <unknown>.
It is possible to turn on / off the lights, but it is (and I'm hoping it's related) incredibly slow to do so. I'm talking about 5-10 seconds before it responds

So I started poking around in the Z-Wave JS Device Database to figure out how to contribute.
So I made a json file for the device using the guidelines there, but it stops there.
I cannot figure out the whole pull request thing on github, and it isn't exactly intuitive.

Can someone help me?
I've uploaded my file here:
https://github.com/GD-Dal/z-wave-js/blob/25ba580b15752bc8b0d9ea48b20377a1689d1d16/namron_dimmer.json
But some entries are unclear to me, and probably wrong as they stay now.
I'm thinking about:
"manufacturerId", productType and productID.
How do I obtain that info ?

Here is a link to the product:
https://www.namron.com/products/namron-z-wave-dimmer-2-400w/
And here is a link to the manual:
https://www.elektroimportoren.no/docs/lib/4512724-Brukerveiledning-5.pdf

**Comment by AlCalzone** [comment]
> "manufacturerId", productType and productID

HA should show you these. If you have access to zwavejs2mqtt, it will show something like "Unknown device 0xabcd:0x1234:0x5678", which is the manufacturerId, productType and productID, separated by `:`.

> I cannot figure out the whole pull request thing on github, and it isn't exactly intuitive.

If you know where to look, it is not that hard. You start by navigating to the correct folder (should be outlined in the docs) and click "Add file", select "Create new file":
![grafik](https://user-images.githubusercontent.com/17641229/116817722-1445cf80-ab68-11eb-8d5e-de1dd7c185a5.png)
Github will guide you through it and you'll end up with a PR.
